Assets/Scenes/AAI3EnvironmentManager.unity

```diff
@@ -38,7 +38,7 @@ RenderSettings:
   m_ReflectionIntensity: 1
```

Comment:
I've removed the redundant UI element (Yaml Information UI) where it would give info on the number of arenas and their respective objects. This was requested to be removed by Lucy as it served no actual use case. Good catch as I thought I mentioned this in the PR.